modules/clients/src/test/java/org/apache/ignite/internal/processors/rest/JettyRestProcessorAbstractSelfTest.java

```diff
@@ -1428,6 +1428,28 @@ public void testTopology() throws Exception {
                 assertEquals(publicCache.getConfiguration(CacheConfiguration.class).getCacheMode(), cacheMode);
             }
         }
+
+        // Test that caches were excluded.
+        ret = content(null, GridRestCommand.TOPOLOGY,
+            "attr", "false",
+            "mtr", "false",
+            "excludeCaches", "true"
```

Comment:
Add test for behavior without excludeCaches parameter (to check backward compatibility).

Reply:
Actually it is tested by first part of this test.